deployments/editor/libero-editor--prod.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     fluxcd.io/automated: "true"
     filter.fluxcd.io/client: semver:*
     filter.fluxcd.io/articleStore: semver:*
+    filter.fluxcd.io/transformer: glob:master-*-*
```

Comment:
Can you create a followup task to switch this to semver rather than glob.

Reply:
Yep, I knew when I put this in that we'd probably want to have a fixed versioning deployment but thought it'd be a good starting point.